backend/src/test/java/com/benine/backend/camera/ipcameracontrol/IpcameraTest.java

```diff
@@ -39,6 +39,43 @@ public final void setUp() throws InvalidCameraTypeException{
 	  camera = new IPCamera("127.0.0.1:9002");
 	  mockServerClient.reset();
   }
+  
+  @Test
+  public final void testGetMACAddress() throws CameraConnectionException {
+    parameterList = new ArrayList<Parameter>();
+    parameterList.add(new Parameter("FILE", "1"));
+
+    final HttpRequest request = HttpRequest.request("/cgi-bin/getinfo")
+                                    .withQueryStringParameters(parameterList);
+    mockServerClient.when(request).respond(HttpResponse.response().withBody("MAC=8C-C1-21-F0-46-C9 "
```

Comment:
Maybe this message can be put in a separate `.txt` file to make the test more readable.